callbacks/review_callbacks.py

```diff
@@ -75,29 +78,53 @@ def input_submit(n_submit, value):
                     dash.html.P(err, className="error-toast-content"),
                 )
 
-    def update_review_status_and_return(value, status, selected_rows):
+    def update_review_status_and_return(value, status, selected_rows, token_data):
+        token_data = token_data or {}
```

Comment:
isn't this parameter passed in? is it going to be None sometimes?

Reply:
it's none when the page first loads and there's nothing in the store